### ISSUE TITLE
Error: Cannot find module '@babel/register'

### ISSUE THREAD

**Issue by retyui** [facebook/fbt#308]
## 🐛 Bug Report

```log
./infrastructure/i18n/bin.sh manifest

Error: Cannot find module '@babel/register'
Require stack:
- /home/i/all_work/thera/mobile-thera-app/node_modules/babel-plugin-fbt/dist/bin/manifest.js
- /home/i/all_work/thera/mobile-thera-app/node_modules/babel-plugin-fbt/dist/bin/manifest.bin.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/home/i/all_work/thera/mobile-thera-app/node_modules/babel-plugin-fbt/dist/bin/manifest.js:35:1)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/i/all_work/thera/mobile-thera-app/node_modules/babel-plugin-fbt/dist/bin/manifest.js',
    '/home/i/all_work/thera/mobile-thera-app/node_modules/babel-plugin-fbt/dist/bin/manifest.bin.js'
  ]
}
```

## To Reproduce

use the latest version of `babel-plugin-fbt`

## Expected behavior

no exceptions when no `@babel/register`

## Link to repo (highly encouraged) or paste

...

## envinfo

```bash
npx: installed 1 in 0.821s

  System:
    OS: Linux 5.4 Linux Mint 20.2 (Uma)
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
    Memory: 2.26 GB / 15.46 GB
    Container: Yes
    Shell: 3.3.1 - /usr/bin/fish
  Binaries:
    Node: 14.18.1 - /usr/bin/node
    Yarn: 1.22.10 - /usr/bin/yarn
    npm: 6.14.15 - /usr/bin/npm
    Watchman: 20210110.135312.0 - /usr/local/bin/watchman
```

**Comment by retyui** [comment]
@pkqinys I think you have other dependencies that depends on `@babel/register`

**Comment by kayhadrin** [comment]
Fixed in 482e18c49090c72e2ccbb542a05f7f09a9612286
